docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -392,7 +392,7 @@ starlight({
 });
 ```
 
-Browsing the documentation in Brazilian Portuguese will generate the following sidebar:
+При просмотре документации на бразильском португальском языке будет сгенерирована следующая боковая панель:
```

Comment:
I think this translation may be a small mistake. Would you mind reverting it?

Reply:
My bad 😥

Reply:
I was confused comparing the old and new translation.

Reply:
No worries at all, tiny mistakes happen to all of us, that's what reviews are for, team work makes the best work 🙌

Reply:
Done

Reply:
Thank you for your support ❤